libs/block-content/website/src/lib/break/break-block.tsx

```diff
@@ -50,12 +44,12 @@ export const BreakBlock = ({
   const reverse = layoutOption === 'image-right'
 
   return (
-    <BreakBlockWrapper className={className} reverse={reverse}>
-      <BreakBlockSegment reverse={reverse}>
-        {image && <Image image={image} square />}
-      </BreakBlockSegment>
+    <BreakBlockWrapper className={className}>
+      <BreakBlockSegment reverse={reverse}>{image && <Image image={image} />}</BreakBlockSegment>
+
       <BreakBlockSegment>
-        <H4 css={headingStyles}>{text}</H4>
```

Comment:
H4 was semtantically wrong there, as it can't be an article semantically, i added the component `div`.